### DISCUSSION TITLE
Building the Application

### DISCUSSION THREAD

**Discussion by sthasam2** [lnreader/lnreader#1146] [Q&A]
Is there a guide on how to build the application? 

I'm a newbie in react native. I wanted to try building the application, but got stuck onto some build issues. 
is there a guide on how to build the application?

**Comment by nyagami** [comment]
There's a guide in the website https://lnreader.github.io/contribute.
But I think it is quite old.
The best way is checking the environment setup from github build action https://github.com/LNReader/lnreader/blob/master/.github/workflows/build.yml
